xml/System.Buffers/SequenceReaderExtensions.xml

```diff
@@ -45,13 +45,13 @@
         <remarks>To be added.</remarks>
       </Docs>
     </Member>
-    <Member MemberName="TryReadBigEndian">
-      <MemberSignature Language="C#" Value="public static bool TryReadBigEndian (this ref System.Buffers.SequenceReader&lt;byte&gt; reader, out int value);" />
-      <MemberSignature Language="ILAsm" Value=".method public static hidebysig bool TryReadBigEndian(valuetype System.Buffers.SequenceReader`1&lt;unsigned int8&gt;&amp; reader, [out] int32&amp; value) cil managed" />
-      <MemberSignature Language="DocId" Value="M:System.Buffers.SequenceReaderExtensions.TryReadBigEndian(System.Buffers.SequenceReader{System.Byte}@,System.Int32@)" />
-      <MemberSignature Language="VB.NET" Value="&lt;Extension()&gt;&#xA;Public Function TryReadBigEndian (ByRef reader As SequenceReader(Of Byte), ByRef value As Integer) As Boolean" />
-      <MemberSignature Language="C++ CLI" Value="public:&#xA;[System::Runtime::CompilerServices::Extension]&#xA; static bool TryReadBigEndian(System::Buffers::SequenceReader&lt;System::Byte&gt; % reader, [Runtime::InteropServices::Out] int % value);" />
-      <MemberSignature Language="F#" Value="static member TryReadBigEndian :  *  -&gt; bool" Usage="System.Buffers.SequenceReaderExtensions.TryReadBigEndian (reader, value)" />
+    <Member MemberName="TryReadLittleEndian">
```

Comment:
same here... there should be three, there's only two here

![image](https://user-images.githubusercontent.com/12971179/61482163-db1a7c00-a94e-11e9-933e-fe2724c8c529.png)

Reply:
This appears to be resolved locally now, as all three overloads are present in ecmaxml